hardhat.config.js

```diff
@@ -4,7 +4,10 @@ require("hardhat-gas-reporter");
 require("solidity-coverage");
 require("dotenv").config();
 
-const { MUMBAI_API_URL, POLYGON_API_URL, PRIVATE_KEY, POLYSCAN_KEY } = process.env;
+const MUMBAI_API_URL = process.env.MUMBAI_API_URL || "";
```

Comment:
I now added defaults for all env variables so we don't need dummy .env files anymore.